modules/cli/src/main/java/org/apache/ignite/cli/call/status/StatusReplCall.java

```diff
@@ -40,9 +39,7 @@
 public class StatusReplCall implements Call<EmptyCallInput, Status> {
 
     private final NodeManager nodeManager;
-
     private final CliPathsConfigLoader cliPathsCfgLdr;
```

Comment:
I don't think that removing the empty lines is a good idea.

Reply:
Code style states that it is optional and I think it's better to use libe breaks between groups of fields, rather than between each field.

Reply:
The whole Ignite 3 source code uses line breaks. I believe it is better not to break the convention even if something looks more readable.